sync-todo/v2/client/cpp/views/scroller.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef SCROLLER_H
+#define SCROLLER_H
```

Comment:
Btw, this is the pattern you "should" use over `#pragma once`, which is not universally supported.